src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -80,8 +98,39 @@ def __init__(self, start: int, end: int):
         )
         self._condition = threading.Condition()
         self._no_remote_peer = 0
+        self._metadata_keys = [key.lower() for key in metadata_keys]
+        self._include_all_metadata = "*" in [
+            key.strip() for key in metadata_keys
+        ]
```

Comment:
would adding `.strip()` to the preprocess make sense?

```suggestion
        self._metadata_keys = [key.strip().lower() for key in metadata_keys]
        self._include_all_metadata = "*" in self._metadata_keys
```